code/datums/setup_option/backgrounds/origin.dm

```diff
@@ -22,14 +22,16 @@
 		STAT_COG = -8
 	)
 
+	if /mob/living/carbon/human
+		add_language(LANGUAGE_GERMAN)
```

Comment:
This part will not compile, remove it please.

Reply:
How does it have to be then? Has the if to be removed to add the language to Oberth origin?
I can't deduce how in the world that should be phrased.

Reply:
It does not need any code, datum are meant to only contain variables for other parts of code to work.

Reply:
This is because the example file does not need it thus this file does not need this check. Also, the reason why it must be removed is the /datum is basically already exposes to the rest of code ready to be used. You can see in https://github.com/discordia-space/CEV-Eris/pull/4095 which he adds only these variables.

Reply:
Okay, thank you for the explanation, i will just remove it then in this case.